src/App.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Kindly refer to the commend above and go with React.PureComponent. equally, check the `Navigation `component